README.md

```diff
@@ -2,7 +2,7 @@ DataFrames.jl
 =============
 
 [![0.6](http://pkg.julialang.org/badges/DataFrames_0.6.svg)](http://pkg.julialang.org/?pkg=DataFrames)
-[![0.7](http://pkg.julialang.org/badges/DataFrames_0.7.svg)](http://pkg.julialang.org/?pkg=DataFrames)
+<!-- [![0.7](http://pkg.julialang.org/badges/DataFrames_0.7.svg)](http://pkg.julialang.org/?pkg=DataFrames) -->
```

Comment:
Why change this?

Reply:
It doesn't render a badge on the README. I think because v0.7 doesn't have tests on pkg.julialang.org [link](https://pkg.julialang.org/?pkg=DataFrames#DataFrames). I'd also like to change the CI tests to allow failures on nightly until a release candidate comes out for v0.7. 👍  or 👎 ? These can all be moved to another PR too

Reply:
Ah, OK. Just remove it then.

Reply:
Ok, I'm going to remove the change here in favor of a badge-specific PR in https://github.com/JuliaData/DataFrames.jl/pull/1253